woocommerce/woocommerce-ios
# Coupons: Move entry point of coupon list from settings to hub menu

Closes: #5763 


### Description

This PR removes the entry point to the coupon list from the Settings screen and adds it to the new hub menu instead.

Additionally, this also updates some colors on the hub menu to match with the design for dark mode. Classes relating to the coupon list are also renamed for simplicity.

### Testing instructions

With feature flag `.couponManagement` off:
- Log in to a valid account
- Select the hub menu tab - notice that there's no item for coupons.
- Select My Store tab > Settings - notice that there's no item for coupons here either.

With feature flag `.couponManagement` on:
- Log in to a valid account
- Select the hub menu tab - notice that there's an item for coupons. Selecting it should navigate to the Coupons list screen.
- Select My Store tab > Settings - notice that there's no item for coupons here either.

### Screenshots

| Light mode | Dark mode |
| ----- | ----- | 
| <img src="https://user-images.githubusercontent.com/5533851/147735672-fa44b81f-8f47-48fe-88c4-fdfff9031f71.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/147735712-e345a3ae-853b-49af-ad05-cdb3d562a874.png" width=320 > |

---
